### DISCUSSION TITLE
Lesson 6 foundryup doesn't installs foundry after killing the terminal

### DISCUSSION THREAD

**Discussion by NeoRusi** [Cyfrin/foundry-full-course-cu#731] [Q&A]
in lesson 6 ive installed foundry but after I kill the terminal foundry should be installed again maybe someone knows how to tinker with this error ?

**Comment by 0xTagger** [comment]
are you using MacOS/Linux or WSL?

**Reply by NeoRusi** [reply]
MacOs

**Comment by theirrationalone** [comment]
Walkout through this [Comment](https://github.com/Cyfrin/foundry-full-course-f23/discussions/585#discussioncomment-6684385) 😉
